src/index.tsx

```diff
@@ -5,13 +5,13 @@ const contextEnabled = Object.prototype.hasOwnProperty.call(React, "createContex
 const hooksEnabled = Object.prototype.hasOwnProperty.call(React, "useMemo") && Object.prototype.hasOwnProperty.call(React, "useCallback");
 
 export interface IPrintContextProps {
-    handlePrint: () => void,
+    handlePrint: (event: React.MouseEvent, lazyOption?: { content: (() => React.ReactInstance) }) => void,
 }
 const PrintContext = contextEnabled ? React.createContext({} as IPrintContextProps) : null;
 export const PrintContextConsumer = PrintContext ? PrintContext.Consumer : () => null;
 
 export interface ITriggerProps<T> {
-    onClick: () => void;
+    onClick: (event?: React.MouseEvent) => void;
```

Comment:
When would we not pass an event?

Reply:
For backwards compatibility, there are use cases like [this one](https://github.com/gregnb/react-to-print/blob/master/examples/FunctionalComponentWithHook/index.tsx#L62) that need to be supported going forward

See excerpt below: 👇🏾👇🏾👇🏾

```jsx
const handlePrint = useReactToPrint();
...
<button onClick={handlePrint}>
    Print using a Functional Component with the useReactToPrint hook
 </button>
```